go.mod

```diff
@@ -35,9 +35,9 @@ require (
 	github.com/imdario/mergo v0.3.12
 	github.com/mattn/go-isatty v0.0.12 // indirect
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/openshift/api v0.0.0-20210817132244-67c28690af52
-	github.com/openshift/client-go v0.0.0-20210730113412-1811c1b3fc0e
-	github.com/openshift/library-go v0.0.0-20210819104210-e14e06ba8d47
+	github.com/openshift/api v0.0.0-20210831091943-07e756545ac1
```

Comment:
a request from me would also be to update `openshift/api` to the latest so we can pick up the changes for the PowerVS platform which will be supported soon(which is this commit: https://github.com/openshift/api/commit/3611f00b94fd7828c1e94f5c54997bc085a74676)